nx584/client.py

```diff
@@ -23,20 +23,22 @@ def list_partitions(self):
         except TypeError:
             return r.json()['partitions']
 
-    def arm(self, armtype='auto'):
+    def arm(self, armtype='auto', partition = 1):
```

Comment:
Proper PEP8 format for this is no space around the equals. Please see the code style in the surrounding code and keep it consistent.